frontend/src/components/dashboard/page_views/PeriodDropdown.vue

```diff
@@ -0,0 +1,91 @@
+<template>
```

Comment:
don't copy/paste - make 1 common component that will be working with just props, and use it here and in Visitors page